ath-container.sh

```diff
@@ -8,8 +8,8 @@ gid=$(id -g)
 tag="jenkins/ath"
 java_version="${java_version:-8}"
 
-docker build --build-arg=uid="$uid" --build-arg=gid="$gid" --build-arg=java_version="$java_version" src/main/resources/ath-container -t "$tag"
+docker build --build-arg=uid="$uid" --build-arg=gid="$gid" src/main/resources/ath-container -t "$tag"
 
 run_opts="--interactive --tty --rm --publish-all --user ath-user --workdir /home/ath-user/ath-sources"
 run_drive_mapping="-v /var/run/docker.sock:/var/run/docker.sock -v $(pwd):/home/ath-user/ath-sources -v ${HOME}/.m2/repository:/home/ath-user/.m2/repository"
-docker run --shm-size 2g $run_opts $run_drive_mapping $tag /bin/bash
+docker run --shm-size 2g $run_opts $run_drive_mapping $tag /bin/bash -c "./set-java.sh $java_version; bash"
```

Comment:
-1 on the approach here. Note the container can / is supposed to be used interactively - not just to invoke the testsuite. IMO we have to separate that, providing a script to switch the java version to be called  at the time `vnc.sh` is.

Reply:
Oh, it is actually never run. Nevermind.